api/endpoints/user.md

```diff
@@ -100,7 +100,9 @@ Collection of all Users.
     + search (optional, string, `mario`) ... String to search for user by firstName, lastName, or email
     + centerId (optional, number, `12`) ... Id of center to query from. If `centerId` is provided `role` is _required_
     + role (optional, string, `operator`) ... Role name to query from. Valid values are: _administrator_, _operator_, _customer_, _manager_, and _technician_. If `role` is provided `centerId` is _required_
-    
+    + siteAdmin (optional, boolean, `true`) ... If true, query will include users with Admin permissions
+    + hasRoom (optional, boolean, `true`) ... If true, query will include users with rooms
+    + hasDefaultVisitorCenter (optional, boolean, `true`) ... If true, query will include users with a default visitor center
 
```

Comment:
The last section of this file... I'm not sure what that is...

```
 ### Hold a Request [GET]
 + Response 200
     [User][]
```

It also looks like there are some things in `UserResource.scala` that aren't in this file yet:
```
/me/reportAuth
/me/supportAuth
/{id}/resetPassword
/{id}/resetPassword
/{id}/permissions GET & PUT
{id}/attachAvatar
username/{username}
```